transformations/element-plus/element-plus-component-name.ts

```diff
@@ -0,0 +1,49 @@
+import wrap from '../../src/wrapAstTransformation'
+import type { ASTTransformation } from '../../src/wrapAstTransformation'
+import type {
+  ImportSpecifier,
+  ImportDefaultSpecifier
+} from 'ast-types/gen/nodes'
+import { getCntFunc } from '../../src/report'
+
+/**
+ * Process component names that have been changed in element plus
+ * @param content
+ */
+export const transformAST: ASTTransformation = ({ root, j }) => {
+  // find element-ui import
+  const elementPlusImport = root.find(j.ImportDeclaration, {
+    source: {
+      value: 'element-plus'
+    }
+  })
+
+  if (elementPlusImport.length) {
+    elementPlusImport.forEach(({ node }) => {
+      let newSpecifier: (ImportSpecifier | ImportDefaultSpecifier)[] = []
+      node.specifiers.forEach(importNode => {
+        if (importNode.type === 'ImportSpecifier') {
+          newSpecifier.push(
+            j.importSpecifier(
+              j.identifier('El' + importNode.local?.name),
```

Comment:
i think before adding 'El' to the header of the component Name you need to determin whether or not the component name starts with 'El'